PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/graph/nodes/StraightSequenceBubble.java

```diff
@@ -37,12 +38,23 @@ public GraphNode copy() {
   public GraphNode copyAll() {
     return new StraightSequenceBubble(this, aligner);
   }
+  
+  @Override
+  public Collection<Integer> getGenomes() {
+    HashSet<Integer> genomeSet = new HashSet<>();
+    for (GraphNode nestedNode : getChildren()) {
+      genomeSet.addAll(nestedNode.getGenomes());
+    }
+    for (GraphNode inEdge : getInEdges()) {
+      genomeSet.addAll(inEdge.getGenomes());
+    }
```

Comment:
Why are the genomes of the inedges needed here? This doens't really seem correct.
If this is really needed you could also let the parent class (Bubble) return a HashSet instead of a Collection (this is a child class of Collection, so it still overrides the method correctly) and call
`HashSet<Integer> genomeSet = super.getGenomes()`
that way you can remove these lines:
```java
for (GraphNode nestedNode : getChildren()) {
   genomeSet.addAll(nestedNode.getGenomes());
}
```
so there will be less code duplication. This is not really important but just looks a bit nicer.